### PR TITLE
[INLONG-7043][Dashboard] Add field prompt for ES sink

### LINKED ISSUE #7043: [Improve][Dashboard] Add field prompt for ES sink
### Description

_No response_

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by bluewang**
### Prepare a Pull Request

- Fixes #7043

### Modifications

image

### CODE REVIEW

**Review by dockerzhang — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -167,6 +167,7 @@
   "meta.Sinks.ES.RetryTimes": "重试次数",
   "meta.Sinks.ES.RetryTimesUnit": "次",
   "meta.Sinks.ES.EsVersion": "Elasticsearch 版本",
+  "meta.Sinks.ES.EsVersionHelp": "此处 Elasticsearch 版本为大版本号",
```
Comment (healchow) [inline inlong-dashboard/src/locales/cn.json:170]: Is the following comment correct?
```suggestion
  "meta.Sinks.ES.EsVersionHelp": "填写大版本号，如：对于7.5.2版本，填写7即可",
```

Comment (dockerzhang) [reply inlong-dashboard/src/locales/cn.json:170]: like 5,6,7

Comment (dockerzhang) [reply inlong-dashboard/src/locales/cn.json:170]: Maybe we can add a drop-down selection box. The options include all supported versions, such as 5/6/7.

Comment (healchow) [reply inlong-dashboard/src/locales/cn.json:170]: This is a good way.
However, there will be too much coupling: if the versions supported by the background become more or less, the front end needs to be modified.

Comment (healchow) [reply inlong-dashboard/src/locales/cn.json:170]: This limitation is recommended to be placed on the Manager side. It is filled in by the user and verified by the Manager module when saving. If there is any error, the error will be thrown directly.

**Review by yunqingmoswu — APPROVED** [review]

**Review by leezng — APPROVED** [review]

File: inlong-dashboard/src/locales/cn.json (hunk 1 above)
Comment (dockerzhang) [reply inlong-dashboard/src/locales/cn.json:170]: We can deal with it simply and only give a clear reminder.

Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:170]:
```suggestion
  "meta.Sinks.ES.EsVersionHelp": "Elasticsearch 大版本号，比如 5/6/7",
```

File: inlong-dashboard/src/locales/en.json [hunk 2]
```diff
@@ -167,6 +167,7 @@
   "meta.Sinks.ES.RetryTimes": "RetryTimes",
   "meta.Sinks.ES.RetryTimesUnit": "items",
   "meta.Sinks.ES.EsVersion": "Elasticsearch Version",
+  "meta.Sinks.ES.EsVersionHelp": "Here the Elasticsearch version is the major version number",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/en.json:170]:
```suggestion
  "meta.Sinks.ES.EsVersionHelp": "Elasticsearch major version number, such as 5/6/7",
```

**Review by healchow — APPROVED** [review]

**Pull request merged**